view/statistics-time-per-person.js

```diff
@@ -39,14 +39,25 @@ var getRowResult = function (rowData, label) {
 	return result;
 };
 
+var processingStepsMetaFrontDeskFilter = function (processingStepsMeta) {
```

Comment:
Yeah, OK I think I ill advised you last time. Let's make a util out of this. PLace it in `view/utils`